security/container_security/security-storage.adoc

```diff
@@ -0,0 +1,28 @@
+[id="security-storage"]
+= Securing attached storage
+include::modules/common-attributes.adoc[]
+:context: security-storage
+
+toc::[]
+
+{product-title} supports multiple types of storage, both
+for on-premise and cloud providers. In particular,
+{product-title} can use storage types that support the Container
+Storage Interface.
+
+// Persistent volume plug-ins
+include::modules/security-storage-persistent.adoc[leveloffset=+1]
+
+// Shared storage
+include::modules/security-storage-shared.adoc[leveloffset=+1]
+
+// Block storage
+include::modules/security-storage-block.adoc[leveloffset=+1]
+
+.Additional resources
+- xref:../../storage/understanding-persistent-storage.adoc#understanding-persistent-storage[Understanding persistent storage]
+- xref:../../storage/container_storage_interface/persistent-storage-csi.adoc#persistent-storage-using-csi[Configuring CSI volumes]
+- xref:../../storage/dynamic-provisioning.adoc#dynamic-provisioning[Dynamic provisioning]
+- xref:../../storage/persistent_storage/persistent-storage-nfs.adoc#persistent-storage-using-nfs[Persistent storage using NFS]
+- xref:../../storage/persistent_storage/persistent-storage-aws.adoc#persistent-storage-using-aws-ebs[Persistent storage using NFS]
```

Comment:
Title needs updated to "Persistent storage Using AWS Elastic Block Store"

Reply:
Oops. Missed fixing a copy and paste. Thanks. Fixed.